dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -125,6 +148,7 @@ window.Dashboard = {
 		if ( !started ) {
 			this.__started = true;
 			this.run();
+			this.__isCertInstalled();
```

Comment:
`__isCertInstalled()` must be called after run(), because we need the dispatcher as well as the client.